src/Tizen.Multimedia.Remoting/MediaController/MediaControllerShuffleMode.cs

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Tizen.Multimedia.Remoting
+{
+    /// <summary>
+    /// Specifies the shuffle mode.
+    /// </summary>
+    /// <since_tizen> 5 </since_tizen>
+    public enum MediaControlShuffleMode
+    {
+        /// <summary>
+        /// Off.
+        /// </summary>
+        Off,
```

Comment:
Why do you reverse the value?

Do you know about that it is not add, it is changed
internal Tizen.Multimedia.Remoting.MediaControllerShuffleMode
->
public Tizen.Multimedia.Remoting.MediaControllerShuffleMode

Reply:
No, It's added in public newly.

As you said above, ```internal Tizen.Multimedia.Remoting.MediaControllerShuffleMode``` exists and it have same order with the enum of native fw.(```mc_shuffle_mode_e```).
But it does not exposed in public, because IsShuffleModeEnabled() and SetShuffleModeEnabled() convert it as ```bool``` type.

Now we added ShuffleModeCommand class and it have Mode property, so I added ```public MediaControlShuffleMode``` enum as same order with ```public MediaControlRepeatMode``` and old ```internal MediaControlShuffleMode``` is changed ```internal MediaControllerNativeShuffleMode```.

Reply:
But It's unlikely that shuffle mode have other value except on, off.
So I'll remove public enum and the ```Mode``` property of ShuffleModeCommand class will be changed ```bool``` type.